setup.py

```diff
@@ -13,8 +13,12 @@
 
 # About dict to store version and package info
 about = dict()
-with open('partridge/__version__.py', 'r', encoding='utf-8') as f:
-    exec(f.read(), about)
+if sys.version_info[0] >= 3:
+    with open('partridge/__version__.py', 'r', encoding='utf-8') as f:
+        exec(f.read(), about)
+else:
+    with open('partridge/__version__.py', 'r') as f:
+        exec(f.read().decode('utf-8'), about)
 
 requirements = [
```

Comment:
We should add functools32 to this on Python 2.

Reply:
I'm not sure I follow this requested change. Can you provide more description?